libnavigation-core/build.gradle

```diff
@@ -54,11 +54,13 @@ dependencies {
     implementation project(':libnavigation-util')
     implementation project(':libnavigator')
     runtimeOnly project(':libdirections-hybrid')
-    runtimeOnly project(':libtrip-notification')
+    implementation project(':libtrip-notification')
```

Comment:
Why do we need to change it?

Reply:
`TripServiceActivity` doesn't compile, because it needs `MapboxTripNotification`

Reply:
By adding `implementation`, we can mistakenly use classes from the implementation package and break modularity.

Would this work?
```suggestion
  runtimeOnly project(':libtrip-notification')
  debugImplementation project(':libtrip-notification')
```

Reply:
Also, if you need the instance of the `MapboxTripNotification`, you can try using the module provider instead.

Reply:
> you can try using the module provider instead

This would be preferable, if possible.

Reply:
fixed